src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/client/AssessmentApiClient.kt

```diff
@@ -16,10 +16,8 @@ import java.time.LocalDateTime
 @Component
 class AssessmentApiClient(@Qualifier("assessmentWebClientAppScope") private val webClient: WebClient) {
 
-  fun getAssessmentAnswers(assessmentId: String?): Collection<Question> {
-    return assessmentId?.let {
-      getAssessmentAnswersCall(assessmentId)
-    } ?: listOf()
+  fun getAssessmentAnswers(assessmentId: String): Collection<Question> {
```

Comment:
Upstream call is never null, surely an assessmentId must be supplied?

Reply:
👍 excellent spot

Reply:
I think assumption  are dangerous ehehhe I would return an empty list in case.